Sources/mParticle-AppsFlyer/MPKitAppsFlyer.m

```diff
@@ -86,6 +86,7 @@ - (MPKitExecStatus *)didFinishLaunchingWithConfiguration:(NSDictionary *)configu
     }
     
     _configuration = configuration;
+    [appsFlyerTracker waitForATTUserAuthorizationWithTimeoutInterval:60];
```

Comment:
In the docs it says: 

> Set timeoutInterval as such that app users have enough time to see and engage with the ATT prompt. A few examples:
>
> If ATT prompt is displayed on app launch–a 60-second interval should be enough
If ATT prompt is displayed after a tutorial that takes approximately 2 minutes to complete–a 120-second interval should be enough.

Is there a need to expose this as an option in some way to the customer then? I believe `requestTrackingAuthorization` is called in the customer's app, so depending on when they call it, we may need a longer timeout right?

Reply:
Other than this question, the implementation looks fine based on the docs.

Reply:
@einsteinx2 Since 95% of apps request at or shortly after launch this seemed the most appropriate since it can block data being sent to appsflyer if it's too long. To allow the client to set this we would need to do a UI change so that it would come down in the config but it seems unnecessary to me.

Reply:
makes sense, approving